indico/modules/designer/templates/template.html

```diff
@@ -14,6 +16,37 @@
 {% endblock %}
 
 {% block content %}
+    {% if tpls_count %}
+        <div class="affected-targets-warning weak-hidden">
+            {% call message_box('warning', fixed_width=true) %}
+                <p>
+                    {%- trans count=tpls_count, strong_start='<strong>'|safe, strong_end='</strong>'|safe,
```

Comment:
I'd put these two as HTML in the translated string. Moving very simple tags like `<strong>` to variables just makes the whole thing less readable with no real gain.